### PR TITLE
Add Article, move scss files, add gem publishing

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -109,7 +109,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```
Comment (payex-dev) [inline Gemfile.lock:112]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368&bid=19143768)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368&bid=19143768)

File: Gemfile.lock (hunk 1 above)
Comment (payex-dev) [inline Gemfile.lock:112]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368)

File: Gemfile.lock [hunk 2]
```diff
@@ -109,7 +120,7 @@ GEM
...
```
Comment (payex-dev) [inline Gemfile.lock:123]: Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5847368&bid=19143768)

**Comment by NullableInt** [comment]
Can't use github hosted gems so I want to merge this to get an official published gem

File: .github/workflows/ruby_push.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  push:
+    branches:
+    - '*'
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
+  build:
+    name: Build, Test and Publish Ruby Gem
```
Comment (asbjornu) [inline .github/workflows/ruby_push.yml:13]:
```suggestion
```

File: .github/workflows/ruby_push.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
+name: Ruby Gem
```
Comment (asbjornu) [inline .github/workflows/ruby_push.yml:1]: Can this file be renamed from `ruby_push.yml` to `gem.yml`?

**Comment by NullableInt** [comment]
> 
> 
> Is `Article no main` a precise title of what this PR does? 🤔

Updated title

**Pull request merged**